builtin/providers/aws/auth_helpers.go

```diff
@@ -75,7 +78,7 @@ func GetAccountId(iamconn *iam.IAM, stsconn *sts.STS, authProviderName string) (
 	}
 
 	if len(outRoles.Roles) < 1 {
-		return "", fmt.Errorf("Failed getting account ID via 'iam:ListRoles': No roles available")
+		return "", errors.New("Failed getting account ID via 'iam:ListRoles': No roles available")
```

Comment:
We are generating a new error rather than wrapping one produced by an upstream API?